fxapom/pages/sign_in.py

```diff
@@ -38,32 +38,20 @@ def __init__(self, base_url, selenium, timeout=TIMEOUT):
 
     def sign_in(self, email, password):
         """Signs in using the specified email address and password."""
-        self.email = email
+        self.type_email(email)
         if self.is_element_visible(*self._next_button_locator):
             self.click_next()
-        self.login_password = password
+        self.type_password(password)
         self.click_sign_in()
 
-    @property
-    def email(self):
-        """Get the value of the email field."""
-        return self.selenium.find_element(*self._email_locator).get_attribute('value')
```

Comment:
@silne30 noticed that the locator here is incorrect and should be `self._email_input_locator`.